projects/discussion-ui/src/lib/wrapper/base-wrapper/base-wrapper.component.html

```diff
@@ -0,0 +1 @@
+<p>base-wrapper works!</p>
```

Comment:
Do we need this component?

Reply:
base wrapper component is needed for creating widgets.